migrations/202008131450.do.nys-settings.sql

```diff
@@ -0,0 +1,2 @@
ackend integration with the ROI to support\\ninteroperability is hosted within the EU and is GDPR compliant. It\\nappears likely that Germany will host a federated server for Europe to\\nsupport European interoperability. Once this becomes available, the\\nbackend infrastructure on AWS will be linked to this server to support\\npan European interoperability.\\n\\n### Appointment of Data Processors\\n\\nAll of the data processors are appointed under Data Processors\\nAgreements in compliance with Article 28 of the GDPR.\\n\\n### Compliance with approved codes of conduct \\n\\nArticle 35 (8) of the GDPR provides that in assessing the impact of\\nprocessing operations, due account should be taken as regards compliance\\nwith approved codes of conduct. Since early April, both the European\\nCommission and the European Data Protection Board (EDPB) have issued\\ndocumentation on the use of mobile apps for the fight against the\\nCOVID-19 pandemic that are relevant for the purpose of this DPIA.\\n\\nThe European Commission issued a recommendation on the 8 April 2020 on a\\ncommon Union toolbox for the use of technology and data to combat and\\nexit from the COVID-19 crisis. The eHealth Network issued\\nrecommendations for a common approach to mobile tracing apps. The app\\nhas been assessed against these recommendations and it complies with all\\nrecommendations set out.\\n\\nThe EDPB issued Guidelines 04/2020 on the use of location data and\\ncontact tracing tools in the context of the COVID-19 outbreak.[^7] While\\nthe guidance is neither prescriptive nor exhaustive and the purpose is\\nto provide general guidance, the app was assessed against the\\nrecommendations and obligations outlined in the documents. The app\\ncomplies with conditions set out therein. The App is a COVID-19 pandemic\\nresponse app, one that only performs contact tracing. The clear\\nalignment of purposes of the app functions with respect to the response\\nto the pandemic.\\n\\nThe detailed assessment of the app against the above guidelines are to\\nbe published online.[^8]\\n\\nIdentify and Assess Risks\\n-------------------------\\n\\nAppendix E sets out the risks that have been identified for the project\\nand the levels for those risks *if not mitigated*. Overall risk score\\nfor each risk identified is calculated as the product of the risk\\nlikelihood score and the risk impact score (i.e. likelihood score X\\nimpact score). The following sets out the metrics used in documenting\\nthe risk assessment.\\n\\nIdentify Measures to Reduce Risks\\n---------------------------------\\n\\nAn evaluation of the identified risks in the previous section has been\\ncarried out and a series of measures have been detailed that seek to\\nmitigate those risks to an acceptable level. The table in Appendix F\\nsets out these mitigation measures and an assessment of the risk impact\\ndue to their introduction. The table also sets out if these mitigation\\nmeasures have been approved.\\n\\n"}}');
```

Comment:
You've got some un-escaped quotes in here which will cause problems. Beyond this, the general approach looks good to me (as long as these NYS specific values don't end up in the upstream repo).

Reply:
The first INSERT statement fails with error 
```
ERROR:  syntax error at or near "Health"
LINE 1: ...hern Ireland (under powers available through the 'Health and...
                                                             ^
SQL state: 42601
Character: 792
```

Reply:
Fixing.

Reply:
I don't think I need the languages quite like that based on what I'm seeing from Ireland's prod instance.

Reply:
Updated and now matching more from prod.